extensions/udapdr/scripts/dsp_utilities/generate_ColBERTv2_zeroshot_results.py

```diff
@@ -0,0 +1,87 @@
+
+import numpy as np
+import os
+import torch
+import tqdm
+import time
+
+from primeqa.ir.dense.colbert_top.colbert.infra import Run, RunConfig, ColBERTConfig
+from primeqa.ir.dense.colbert_top.colbert.data import Queries
+
+from primeqa.ir.dense.colbert_top.colbert import Indexer, Searcher
+
+#############################################################################
+
+def evaluate(CKPT, insert_experiment_name_here, chosen_collection, chosen_queries, index=True):
+    split = "dev"
+    nbits = 2
+    k = 1000
+    collection = chosen_collection
+    queries = chosen_queries
+    experiment = (f"{insert_experiment_name_here}.nbits={nbits}",)
+
+    if not os.path.exists(collection):
+        print(f"No data found for {dataset} at {collection}, skipping...")
+        return
+    with Run().context(RunConfig(nranks=1)):
+        INDEX_NAME = f"{insert_experiment_name_here}.nbits={nbits}.latest"
+
+        if index:
+            config = ColBERTConfig(
+                doc_maxlen=220,
+                query_maxlen=32,
+                nbits=nbits,
+                kmeans_niters=4,
+                root="/lfs/1/jonsf/colbert/experiments",
```

Comment:
Avoid the pathname string, perhaps using a similar construction as with `INDEX_NAME` on line 27?